EngineLayer/CommonParameters.cs

```diff
@@ -51,7 +51,8 @@ public CommonParameters()
             ListOfModsFixed = listOfModsFixed ?? new List<(string, string)> { ("Common Fixed", "Carbamidomethyl on C"), ("Common Fixed", "Carbamidomethyl on U") };
             DissociationType = dissociationType;
             SeparationType = separationType;
-            ChildScanDissociationType = childScanDissociationType;
+            MS2ChildScanDissociationType = ms2childScanDissociationType;
```

Comment:
all these dissociation types are getting to be a mess. maybe we should think about a less messy solution. Perhaps a collection of dissociation types?

Reply:
This covers all the situations. I had a disscussion with Rob today. We think it is OK to keep it like this right now.

Reply:
we need a long term solution to the problem of keeping track of what dissociation types are where.  see: https://github.com/smith-chem-wisc/MetaMorpheus/issues/1834